packages/server/db/users.js

```diff
@@ -101,10 +101,10 @@ const createUser = async (uid, body) => {
     const parent = await parseParent(body);
     batch.set(parentsCollectionRef.doc(uid), parent);
 
-    body.students.forEach(async (student) => {
-      const newStudent = await parseStudent(student);
+    const validatedStudents = await Promise.all(body.students.map((s) => parseStudent(s)));
+    validatedStudents.students.forEach((student) => {
```

Comment:
Need to do async stuff outside of loop since `forEach` won't actually block during the await

Reply:
Ah, I see. Thanks!